tabs/osd.js

```diff
@@ -420,6 +432,12 @@ OSD.chooseFields = function () {
         F.PID_YAW,
         F.POWER
       ]);
+      if (semver.gt(CONFIG.flightControllerVersion, "3.1.0")) {
```

Comment:
I'd rather check `semver.gte(CONFIG.apiVersion, "1.32.0")`, since api version is already set for the next release, no matter if that will be 3.1.1, 3.2.0, or an RC of one of these.